src/list.c

```diff
@@ -108,6 +109,11 @@ static void __exit custom_exit(void)
 	struct list_head *ptr, *next;
 	struct identity *entry;
 
+	/*
+	 * here the list is empty, right?
+	 * due to identity_destroy(2); identity_destroy(1); in module_init-function
+	 * so nothing will be kfree-ed..
+	 */
```

Comment:
list entries were removed in module_init-function, so memory will not be kfree-ed

Reply:
True... I guess I don't need that part at all